Scripts/RebuildMids.bat

```diff
@@ -0,0 +1,7 @@
+cd ..
+cd DRODUtil
+cd Release
+DRODUtil.exe import -t ../../Texts/
```

Comment:
Does the '-t' option rebuild mids.h from scratch?

Reply:
You tell me, I just experimented to get it to work at all! I think it might be better to just merge this fil as-is and you can update it to match whatever you do in case you need new MIDs?

Reply:
Ah, my apologies for the presumption.

Here is how you rebuild mids.h and the main .dat file whenever you add new MIDs:
```
drodutil delete
drodutil create
drodutil import <path to Texts dir>
```